src/bitops.c

```diff
@@ -669,15 +662,10 @@ void bitopCommand(client *c) {
     if (maxlen) {
         res = (unsigned char*) sdsnewlen(NULL,maxlen);
         unsigned char output, byte;
+        unsigned long srcbuf[4], dstbuf[4];
         unsigned long i;
 
-        /* Fast path: as far as we have data for all the input bitmaps we
-         * can take a fast path that performs much better than the
-         * vanilla algorithm. On ARM we skip the fast path since it will
-         * result in GCC compiling the code using multiple-words load/store
-         * operations that are not supported even in ARM >= v6. */
         j = 0;
-        #ifndef USE_ALIGNED_ACCESS
```

Comment:
Unaligned load. Fix is to copy variables to a temporary aligned array, execute operation and store back variables to destination. Makes code a bit uglier.